# These two patches are to reduce the --fflags and --flibs output of nf-config to only what is necessary
#
--- nf-config.in.orig	2023-05-19 22:19:16
+++ nf-config.in	2023-09-19 18:40:48
@@ -11,14 +11,14 @@
 
 cc="@CC@"
 fc="@FC@"
-cflags="-I${includedir} @CPPFLAGS@" 
-fflags="-I${includedir} @MOD_FLAG@${fmoddir}"
+cflags="-I${includedir}"
+fflags="-I${includedir}"
 has_dap="@HAS_DAP@"
 has_nc2="@HAS_NC2@"
 has_nc4="@HAS_NC4@"
 has_f90="@HAS_F90@"
 has_f03="@HAS_F03@"
-flibs="-L${libdir} @NC_FLIBS@"
+flibs="-L${libdir} -lnetcdff"
 version="@PACKAGE_NAME@ @PACKAGE_VERSION@"
 
 usage()
--- nf-config.cmake.in.orig	2023-05-19 22:19:16
+++ nf-config.cmake.in	2023-09-19 18:40:21
@@ -12,15 +12,15 @@
 #
 cc="@CMAKE_C_COMPILER@"
 fc="@CMAKE_Fortran_COMPILER@"
-cflags="-I@CMAKE_INSTALL_PREFIX@/include @CMAKE_C_FLAGS@ @CMAKE_CPP_FLAGS@"
-fflags="-I${includedir} @MOD_FLAG@${fmoddir}"
+cflags="-I${includedir}"
+fflags="-I${includedir}"
 #
 has_dap="@HAS_DAP@"
 has_nc2="@HAS_NC2@"
 has_nc4="@HAS_NC4@"
 has_f90="@HAS_F90@"
 has_f03="@HAS_F03@"
-flibs="-L${libdir} @NC_FLIBS@"
+flibs="-L${libdir} -lnetcdff"
 version="@PACKAGE_NAME@ @PACKAGE_VERSION@"
 
  usage()
